.github/workflows/license_check.yml

```diff
@@ -0,0 +1,56 @@
+name: Check licenses
+
+on: 
+  workflow_dispatch:
+  push:
+    paths-ignore:
+      - README.md
+
+jobs:
+  prebuild-job:
+    name: Prebuild Job
+    uses: th2-net/.github/.github/workflows/compound-prebuild-java-dev-workflow.yml@main
+    with:
+      runsOn: 'ubuntu-latest'
```

Comment:
issue: Because this is a reusable workflow now, it does not need this anymore, I think. The version should be passed from outside as an input parameter (e.g. [here](https://github.com/th2-net/.github/blob/th2-1836-json-files-update/.github/workflows/compaund-java-multi-project-build.yml#L4)).

Reply:
fixed